job-server/src/main/scala/spark/jobserver/WebApi.scala

```diff
@@ -908,7 +917,7 @@ class WebApi(system: ActorSystem,
                             case _: MalformedURLException => completeWithException(
                               ctx, "JOB LOADING FAILED: Malformed URL", StatusCodes.BadRequest, err)
                             case _ => completeWithException (
-                              ctx, "JOB LOADING FAILED", StatusCodes.BadRequest, err)
+                              ctx, "JOB LOADING FAILED", StatusCodes.InternalServerError, err)
```

Comment:
Why do you think that InternalServerError is better?
Job loading error usually shows that request was incorrect, some problem with a job.

It feels a bit unconnected to the PR (POST /jobs endpoint) and it changes REST API responses a bit (which should be reflected in the https://github.com/spark-jobserver/spark-jobserver/blob/master/doc/WebApi/swagger.yaml#L405)

Reply:
Hm, I can't recall why I changed it in the first place. I will revert it tomorrow.

Reply:
Ok, it is gone now.